### PR TITLE
Roary: Copy input files instead of symlinking.

### PR DESCRIPTION
**Pull request by davebx**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR updates an existing tool or tool collection

Roary resolves symlinks and tries to act on the input dataset directly, which causes issues if the path contains a space. With this change, the inputs are copied to a filename without spaces in the job working directory.

### CODE REVIEW

File: tools/roary/roary.xml [hunk 1]
```diff
@@ -14,10 +14,9 @@
     <command detect_errors="exit_code"><![CDATA[
         #import re
         #set $filenames = list()
-        #for $gff in $gff_input.gffs
-            #set escaped_element_identifier = re.sub('[^\w\-]', '_', str($gff.element_identifier))
-            ln -s '$gff' '${escaped_element_identifier}.gff' &&
-            #set $filename = str($escaped_element_identifier) + '.gff'
+        #for $idx, $gff in enumerate($gff_input.gffs)
```
Comment (bgruening) [inline tools/roary/roary.xml:17]: @Slugger70 do we needed the filename here? I remember that users needed it for something - that's why we used the elment_identifier here, no?

Comment (davebx) [reply tools/roary/roary.xml:17]: Ah, I wasn't aware that it might be needed, I can change it back easily enough if Simon confirms that.

**Comment by bgruening** [comment]
tests are failing

File: tools/roary/roary.xml [hunk 2]
```diff
@@ -15,9 +15,8 @@
         #import re
         #set $filenames = list()
         #for $gff in $gff_input.gffs
-            #set escaped_element_identifier = re.sub('[^\w\-]', '_', str($gff.element_identifier))
-            ln -s '$gff' '${escaped_element_identifier}.gff' &&
-            #set $filename = str($escaped_element_identifier) + '.gff'
+            #set $filename = '%s.gff' % re.sub('[^\w_-]', '_', str($gff.element_identifier))
```
Comment (bernt-matthias) [inline tools/roary/roary.xml:18]: Maybe add a comment why this copied. Can you open an issue upstream?

File: tools/roary/.shed.yml [hunk 3]
```diff
@@ -6,4 +6,5 @@ long_description: |
 name: roary
 owner: iuc
 remote_repository_url: https://github.com/galaxyproject/tools-iuc/tree/master/tools/roary
+homepage_url: https://toolshed.g2.bx.psu.edu/view/iuc/roary
```
Comment (bernt-matthias) [inline tools/roary/.shed.yml:9]: Shouldn't this be https://sanger-pathogens.github.io/Roary/?

File: tools/roary/roary.xml [hunk 4]
```diff
@@ -55,7 +54,7 @@
         <param name="percent_ident" type="integer" value="95" label="minimum percentage identity for blastp" help="Sets the minimum percentage identity for protein matches" />
         <param name="core_diff" type="float" value="99.0" label="percentage of isolates a gene must be in to be core" help="The total percentage of the isolates that must have the gene for it to be considered a core gene." />
 
-        <param name="outputs" type="select" multiple="true" display="checkboxes" label="Additional outputs" help="Summary statistics, core gene alignments and Gene Presence/Absence tables are always produced.">
+        <param name="outputs" type="select" multiple="true" label="Additional outputs" help="Summary statistics, core gene alignments and Gene Presence/Absence tables are always produced.">
```
Comment (bernt-matthias) [inline tools/roary/roary.xml:57]: This is a bug in the linter, see https://github.com/galaxyproject/galaxy/pull/12920. So you may leave this as is.

Comment (davebx) [reply tools/roary/roary.xml:57]: Good to know, thanks

File: tools/roary/.shed.yml (hunk 3 above)
Comment (davebx) [reply tools/roary/.shed.yml]: Ah yes, so it should

**Review by bernt-matthias — APPROVED** [review]

**Pull request merged**